org.openhab.binding.zigbee/README.md

```diff
@@ -10,6 +10,19 @@ A ZigBee Coordinator is the network controller, and is therefore the heart of th
 
 Coordinators need to be installed manually and the serial port must be set.
 
+#### Examples for coordinators
+
+| Name and Link              | Coordinator | Comment                       |
+|----------------------------|-------------|-------------------------------|
+|[Texas Instruments CC2531EMK](http://www.ti.com/tool/cc2531emk)|[TI2531](#ti2531-coordinator)|-|
+|A cheap CC2530, CC2531, CC2538 or CC2650 Stick from eBay, Aliexpress, etc. like [this](https://de.aliexpress.com/item/Drahtlose-Zigbee-CC2531-Sniffer-software-protokoll-analyse-Bareboard-Paket-Protokoll-Analyzer-Modul-Usb-schnittstelle-Dongle-Erfassen/32852226435.html) and a module to flash the firmware like [this](https://de.aliexpress.com/item/SmartRF04EB-CC1110-CC2530-ZigBee-Module-USB-Downloader-Emulator-MCU-M100/32673666126.html)|[TI2531](#ti2531-coordinator)|-|
+|[Nortek Security & Control HUSBZB-1](https://nortekcontrol.com/products/2gig/husbzb-1-gocontrol-quickstick-combo/)|[TI2531](#ti2531-coordinator)|Stick contains both Z-Wave and Zigbee|
```

Comment:
This is incorrect - the HUSBZB contains the Ember chipset.

Reply:
You are right, thanks. Mixed up 

HUSBZB-1 | Yes (EZSP)

with `ZNP` for TI.